### ISSUE TITLE
Submitting a simple form shows PHP warning and won't save submission

### ISSUE THREAD

**Issue by shoored** [verbb/formie#322]
**Description**
When submitting a form I get this PHP warning and Formie stops right there.
`strstr(): Empty needle`

**Steps to reproduce**
1. Install Formie
2. Create first form with 2 non required fields. Single-line Text and Email
3. Submit the form
4. No profit

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.13
- Craft version: 3.5.18
- Multi-site: No
- Environment: Valet
- PHP version: 7.4

**Additional context**
Just trying a new feature where I'm using Formie for the first time. It's a default installation, no custom stuff yet.

**Comment by engram-design** [comment]
Are you able to take a screenshot of the error with `devMode` on? Or, have a look in your `storage/logs/formie.log` file for the full error? Just trying to isolate the error.

**Comment by shoored** [comment]
So sorry, meant to do this with the bug report. Duh. 

There isn't even a Formie log file. That's strange.

`
2021-01-27 09:10:06 [-][1][3pbmcbh0fhdlo1tpri6n2ialno][error][yii\base\ErrorException:2] yii\base\ErrorException: strstr(): Empty needle in /Users/sjoerd/Code/mprovement/vendor/verbb/formie/src/services/Submissions.php:434
Stack trace:
#0 /Users/sjoerd/Code/mprovement/vendor/craftcms/cms/src/web/ErrorHandler.php(76): yii\base\ErrorHandler->handleError(2, 'strstr(): Empty...', '/Users/sjoerd/C...', 434)
#1 [internal function]: craft\web\ErrorHandler->handleError(2, 'strstr(): Empty...', '/Users/sjoerd/C...', 434, Array)
#2 /Users/sjoerd/Code/mprovement/vendor/verbb/formie/src/services/Submissions.php(434): strstr('', '')
#3 /Users/sjoerd/Code/mprovement/vendor/verbb/formie/src/controllers/SubmissionsController.php(454): verbb\formie\services\Submissions->spamChecks(Object(verbb\formie\elements\Submission))
#4 [internal function]: verbb\formie\controllers\SubmissionsController->actionSubmit()
#5 /Users/sjoerd/Code/mprovement/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#6 /Users/sjoerd/Code/mprovement/vendor/yiisoft/yii2/base/Controller.php(180): yii\base\InlineAction->runWithParams(Array)
#7 /Users/sjoerd/Code/mprovement/vendor/craftcms/cms/src/web/Controller.php(190): yii\base\Controller->runAction('submit', Array)
#8 /Users/sjoerd/Code/mprovement/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('submit', Array)
#9 /Users/sjoerd/Code/mprovement/vendor/craftcms/cms/src/web/Application.php(274): yii\base\Module->runAction('formie/submissi...', Array)
#10 /Users/sjoerd/Code/mprovement/vendor/craftcms/cms/src/web/Application.php(577): craft\web\Application->runAction('formie/submissi...', Array)
#11 /Users/sjoerd/Code/mprovement/vendor/craftcms/cms/src/web/Application.php(253): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#12 /Users/sjoerd/Code/mprovement/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#13 /Users/sjoerd/Code/mprovement/web/index.php(21): yii\base\Application->run()
#14 /Users/sjoerd/.composer/vendor/laravel/valet/server.php(219): require('/Users/sjoerd/C...')
#15 {main}
`

![image](https://user-images.githubusercontent.com/26165948/105961716-2999f200-607f-11eb-8e08-db7ef0c6a6fb.png)

**Comment by engram-design** [comment]
Should be fixed in the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.13",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.14](https://github.com/verbb/formie/releases/tag/1.3.14)
